deepdish/io/hdf5io.py

```diff
@@ -270,7 +280,17 @@ def _load_pickled(level):
 
 def _load_level(handler, level):
     if isinstance(level, tables.Group):
-        dct = {}
+        if _sns and (level._v_title.startswith('sns:') or
+                     DEEPDISH_IO_ROOT_IS_SNS in level._v_attrs):
+            val = sns()
+            dct = val.__dict__
+        elif level._v_title.startswith('list:'):
+            dct = {}
+            val = []
```

Comment:
Is this really necessary for lists? Why not use `lst` as before? This seems like an overcomplication, since lists never actually end up using `dct`.